src/frontend/src/components/mainWindow.ts

```diff
@@ -30,6 +30,16 @@ export const mainWindow = ({
   additionalContainerClasses?: string[];
   HTMLwrapperTag?: string;
 }): TemplateResult => {
+  const pageContent = document.querySelector("#pageContent");
+  const wrap =
+    pageContent !== null ? pageContent : document.querySelector("body");
```

Comment:
I think we use `nonNullish` now instead of explicitly checking for null/undefined